containers/redis/redis.conf

```diff
@@ -169,7 +169,7 @@ loglevel notice
 # Specify the log file name. Also the empty string can be used to force
 # Redis to log on the standard output. Note that if you use standard
 # output for logging but daemonize, logs will be sent to /dev/null
-logfile /var/log/redis/redis-server.log
+logfile
```

Comment:
This causes mayhem with the CS docker setup.
```
*** FATAL CONFIG FILE ERROR (Redis 7.0.7) ***
2022-12-23T17:08:46.215735967Z Reading the configuration file, at line 172
2022-12-23T17:08:46.215738048Z >>> 'logfile'
2022-12-23T17:08:46.215740236Z wrong number of arguments
```

No string != empty string, fellas..